tests/instancetype_test.go

```diff
@@ -397,6 +398,40 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 			Expect(vmi.Annotations).To(HaveKeyWithValue("preferred-annotation-1", "1"))
 			Expect(vmi.Annotations).To(HaveKeyWithValue("preferred-annotation-2", "2"))
 		})
+
+		FIt("[test_id:CNV-10328] should apply preference with PreferredStorageClassName to VM", func() {
+
+			StorageClassName := getAvailableStorageClass(virtClient)
+
+			clusterPreference := newVirtualMachineClusterPreference()
+			clusterPreference.Spec.Volumes = &instancetypev1beta1.VolumePreferences{
+				PreferredStorageClassName: StorageClassName,
+			}
+			clusterPreference, err := virtClient.VirtualMachineClusterPreference().
+				Create(context.Background(), clusterPreference, metav1.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			dataVolume := libdv.NewDataVolume(
+				libdv.WithRegistryURLSource(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskCirros)),
+				libdv.WithPVC(libdv.PVCWithStorageClass(StorageClassName)),
```

Comment:
When you set the StorageClass here, the preference will not overwrite it.
```suggestion
				libdv.WithPVC(),
```